jackmatchmaker/jacklib.py

```diff
@@ -1243,9 +1244,27 @@ def get_time():
 # -------------------------------------------------------------------------------------------------
 # Misc
 
+global _error_callback
```

Comment:
The global declaration is not needed here. Variables in the module level are already module globals by definition.

(I know there are other places in the module where this is used, this is a left-over form the origins of the module in falkTX's Cadence. I will clean these up now.)

Reply:
Yeah that's why I put it here even though that was weird. Removed.